tests/backend/test_logger.py

```diff
@@ -0,0 +1,65 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+from typing import Mapping, Optional
+from hypercorn.config import Config as HyperConfig
```

Comment:
we divide import in 3 category (each separated by an empty line)

1- Python std lib and 3rd party imports
2 - Parsec imports
3 - Tests imports

so here it should be:
```python

from typing import Mapping, Optional
import base64
from hypercorn.config import Config as HyperConfig
from hypercorn.typing import ResponseSummary, HTTPScope

from parsec.backend.asgi.logger import ParsecLogger
```

Reply:
Done.